Fakebook.Profile/Fakebook.Profile.UnitTests/APITests/ContextFactory.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Fakebook.Profile.DataAccess;
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+
+namespace Fakebook.Profile.UnitTests.APITests
+{
+    public class ContextFactory : IDisposable
+    {
+        private DbConnection _conn;
+        private bool disposedValue;
+
+        private DbContextOptions<DataAccess.EntityModel.ProfileDbContext> CreateOptions()
+        {
+            return new DbContextOptionsBuilder<DataAccess.EntityModel.ProfileDbContext>().UseSqlite(_conn).Options;
+        }
+
+        public DataAccess.EntityModel.ProfileDbContext CreateContext()
+        {
+            if (_conn == null)
+            {
+                _conn = new SqliteConnection("DataSource=:memory:");
+                _conn.Open();
+
+                DbContextOptions<DataAccess.EntityModel.ProfileDbContext> options = CreateOptions();
```

Comment:
i think you can reuse the same options instance without issues, shouldn't need to recreate it.

Reply:
modified if statement so we only have to create options once